src/main/java/com/github/lunatrius/schematica/world/schematic/SchematicClassic.java

```diff
@@ -2,20 +2,20 @@
 
 import net.minecraft.nbt.NBTTagCompound;
 
-import com.github.lunatrius.schematica.api.ISchematic;
+import org.apache.commons.lang3.NotImplementedException;
 
-import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+import com.github.lunatrius.schematica.api.ISchematic;
 
-// TODO: http://minecraft.gamepedia.com/Data_values_%28Classic%29
+// TODO: https://web.archive.org/web/20170708003352/http://minecraft.gamepedia.com/Data_values_(Classic)
 public class SchematicClassic extends SchematicFormat {
```

Comment:
In the spirit of cleanup can't this entire class be removed? 
It doesn't seem to do anything.

Reply:
it seems to have never been implemented so I'll remove the class once I'm done with work